platform/security/platform-security-core-api/pom.xml

```diff
@@ -225,6 +225,7 @@
                         </Export-Package>
                         <Import-Package>
                             org.opensaml.xmlsec.signature.support.provider,
```

Comment:
should we have a version here?

Reply:
Next chance I get, I'll spin up a container and see what version is being wired up here.

Reply:
Strange. It's getting wired up correctly without the range. 
```
Import-Package =
	org.opensaml.xmlsec.signature.support.provider,
	org.opensaml.core.config;version=3.3.0,
...
```
Probably because this is the only consumer. I think this will be fine.